src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,47 @@
+import {
+  AuthIcon,
+  CartIcon,
+  HomeIcon,
```

Comment:
You could have included all these icons in one file and then imported them with "* as ALL " and used it with <ALL.ComponentName/>